crane/cmd.go

```diff
@@ -42,8 +43,11 @@ func configCommand(wrapped func(config Config), forceOrder bool) func(cmd *cobra
 			}
 		}
 
-		// Set target from args
-		options.target = args
+		// Set target and potentially command from args
+		if len(args) > 0 {
+			options.target = strings.Split(args[0], "+")
```

Comment:
I don't want to drop the default group, I use that a lot. I might be missing something very obvious, but why does this prevent a default group? I guess it only prevents you from running a command against the default group without explicitly typing `default`. But you could still say `crane run`.